Buildings/Fluid/HeatExchangers/AirToAirHeatRecovery/WheelWithBypassDamper.mo

```diff
@@ -0,0 +1,306 @@
+    effCal(
+    final epsSenCoo_nominal=epsSenCoo_nominal,
+    final epsLatCoo_nominal=epsLatCoo_nominal,
+    final epsSenCooPL=epsSenCooPL,
+    final epsLatCooPL=epsLatCooPL,
+    final epsSenHea_nominal=epsSenHea_nominal,
+    final epsLatHea_nominal=epsLatHea_nominal,
+    final epsSenHeaPL=epsSenHeaPL,
+    final epsLatHeaPL=epsLatHeaPL,
+    final VSup_flow_nominal=m1_flow_nominal/1.293)
+    "Calculates the effectiveness of heat exchange"
+    annotation (Placement(transformation(extent={{-100,0},{-80,20}})));
+  Buildings.Fluid.Sensors.VolumeFlowRate VExh_flow(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal)
+    "Exhaust air volume flow rate sensor"
+    annotation (Placement(transformation(extent={{-40,-30},{-60,-10}})));
+  Buildings.Fluid.Sensors.VolumeFlowRate VSup_flow(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal)
+    "Supply air volume flow rate sensor"
+    annotation (Placement(transformation(extent={{-80,30},{-60,50}})));
+  Buildings.Fluid.Actuators.Dampers.Exponential bypDamSup(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    final dpDamper_nominal=dp1_nominal)
+    "Supply air bypass damper"
+    annotation (Placement(transformation(extent={{-38,70},{-18,90}})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damSup(
+    redeclare package Medium = Medium1,
+    final m_flow_nominal=m1_flow_nominal,
+    final dpDamper_nominal=dp1_nominal)
+    "Supply air damper"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},rotation=0,origin={-32,40})));
+  Buildings.Fluid.Actuators.Dampers.Exponential damExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
+    "Exhaust air damper"
+    annotation (Placement(transformation(
+        extent={{10,10},{-10,-10}},rotation=-90,origin={60,-40})));
+  Buildings.Fluid.Actuators.Dampers.Exponential bypDamExh(
+    redeclare package Medium = Medium2,
+    final m_flow_nominal=m2_flow_nominal,
+    final dpDamper_nominal=dp2_nominal)
+    "Exhaust air bypass damper"
+    annotation (Placement(transformation(extent={{-20,-70},{-40,-50}})));
+  Modelica.Blocks.Sources.RealExpression TSup(
+    final y=Medium1.temperature(
+      Medium1.setState_phX(
+        p=port_a1.p,
+        h=inStream(port_a1.h_outflow),
+        X=inStream(port_a1.Xi_outflow))))
+    "Supply air temperature"
+    annotation (Placement(transformation(extent={{-160,-30},{-140,-10}})));
+  Modelica.Blocks.Sources.RealExpression TExh(
+    final y=Medium2.temperature(
+      Medium2.setState_phX(
+        p=port_a2.p,
+        h=inStream(port_a2.h_outflow),
+        X=inStream(port_a2.Xi_outflow))))
+    "Exhaust air temperature"
+    annotation (Placement(transformation(extent={{-160,-50},{-140,-30}})));
+  Modelica.Blocks.Sources.RealExpression PEle(
+    final y=P_nominal)
```

Comment:
Is it correct? With this calculation, the power consumption will be constant and always equal to the nominal value.